modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/dataset_display/DatasetDisplay.js

```diff
@@ -45,9 +44,7 @@ function DatasetDisplay(props) {
 	const [views, updateViews] = useState(props.views);
 	const [loading, setLoading] = useState(false);
 	const [actionLoading, setActionLoading] = useState(false);
-	const [datasetDisplaySupportSidePanelId] = useState(
-		props.sidePanelId || 'support-side-panel-' + getRandomId()
-	);
+	const [datasetDisplaySupportSidePanelId] = useState(props.sidePanelId);
```

Comment:
It is a little odd that you would use `useState` for something that you never change. If you just want to associate a value once with a component and never update it, `useRef` is probably the right tool for that (in fact, IIRC, if you look at the implementation of `useState` in React, you'll see that it is using refs under the covers to associate data with components).